proto/timecraft/server/v1/timecraft.proto

```diff
@@ -90,17 +90,39 @@ message ProcessIDResponse {
   string process_id = 1;
 }
 
+message SpawnRequest {
+  ModuleSpec module = 1;
+}
+
+message SpawnResponse {
+  string process_id = 1;
+  string ip_address = 2;
```

Comment:
Not sure if we want to return both the ipv4 and ipv6 address when calling spawn, or whether just the ipv4 address is fine?

Reply:
Related to my other comment, I think we'll want to have a more fleshed out description of the process configuration, including the list of network interfaces and their addresses.